recipe/meta.yaml

```diff
@@ -41,6 +41,10 @@ requirements:
     - {{ compiler('cuda') }}  # [linux64 or ppc64le or win]
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - sysroot_linux-64 2.17  # [linux64]
+    - libcufft-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
+    - cuda-profiler-api  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
+    - cuda-nvrtc-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
+    - cuda-driver-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
```

Comment:
Aren't these meant to be in the `host` section? The linking analysis logs show warnings of some libraries missing because they are probably not exporting the runtime deps from `build` (but they would from `host`). Try moving this block to `host`.

Reply:
You are right! Going to do that now, thanks for catching that!

Do you think it is worth setting the runtime deps to be the non-dev versions?

Reply:
Shouldn't be needed as they should all have `run_exports`